src/bid/bid.rs

```diff
@@ -9,9 +9,15 @@ use dusk_plonk::jubjub::{
     AffinePoint, GENERATOR_EXTENDED, GENERATOR_NUMS_EXTENDED,
 };
 use dusk_plonk::prelude::*;
+use kelvin::{ByteHash, Content, Sink, Source};
 use poseidon252::cipher::PoseidonCipher;
+use poseidon252::cipher::ENCRYPTED_DATA_SIZE;
 use poseidon252::sponge::sponge::sponge_hash;
 use rand_core::{CryptoRng, RngCore};
+use std::convert::TryFrom;
+use std::io::{self, Read, Write};
+
+pub const BID_SIZE: usize = ENCRYPTED_DATA_SIZE + 64 + 32 * 5;
```

Comment:
Maybe add documentation to explain this formula? The meaning isn't obvious

Reply:
Right. Done!